ext/standard/math.c

```diff
@@ -47,30 +47,29 @@ static inline double php_intpow10(int power) {
 }
 /* }}} */
 
-#define PHP_ROUND_BASIC_EDGE_CASE() do {\
-		if (places > 0) {\
-			edge_case = fabs((integral + copysign(0.5, integral)) / exponent);\
-		} else {\
-			edge_case = fabs((integral + copysign(0.5, integral)) * exponent);\
-		}\
-	} while (0)
-#define PHP_ROUND_ZERO_EDGE_CASE() do {\
-		if (places > 0) {\
-			edge_case = fabs((integral) / exponent);\
-		} else {\
-			edge_case = fabs((integral) * exponent);\
-		}\
-	} while (0)
+static zend_always_inline double php_round_get_basic_edge_case(const double integral, const double exponent, const int places)
+{
+	return (places > 0)
+		? fabs((integral + copysign(0.5, integral)) / exponent)
+		: fabs((integral + copysign(0.5, integral)) * exponent);
+}
+
+static zend_always_inline double php_round_get_zero_edge_case(const double integral, const double exponent, const int places)
+{
+	return (places > 0)
+		? fabs((integral) / exponent)
+		: fabs((integral) * exponent);
+}
 
 /* {{{ php_round_helper
 	   Actually performs the rounding of a value to integer in a certain mode */
-static inline double php_round_helper(double integral, double value, double exponent, int places, int mode) {
+static inline double php_round_helper(double integral, const double value, const double exponent, const int places, const int mode) {
```

Comment:
`const` for non pointer parameters doesn't do anything as far as I know.

Reply:
I don't even know why I decided to do this... I might have been half asleep...

Reply:
I fixed it!